core/rlp_test.go

```diff
@@ -1,201 +0,0 @@
-// Copyright 2019 The go-ethereum Authors
```

Comment:
Did this file get deleted in upstream?

Reply:
If I'm not mistaken it was deleted by us previously, not sure why it's showing up here. I may have accidentally added it and re-deleted it.

Reply:
This file was added in upstream during this commit, and is still in upstream, it should be re-added unless there's a specific reason why the test is no longer required.

Reply:
Yeap, I readded it and modified it a bit to work